manage.py

```diff
@@ -1,20 +1,28 @@
-from configobj import ConfigObj
+import configparser
+import json
+
 from flask_script import Manager, Server, Shell
 
+from profiles.config import create_config
 from profiles.factory import create_app
+from profiles.models import Client, Clients
+
+config = configparser.ConfigParser()
+config.read('app.cfg')
 
-config = ConfigObj('app.cfg')
+clients_json = json.load(open('clients.json'))
+clients = Clients([Client(name, **clients_json[name]) for name in clients_json])
 
-app = create_app(config)
+app = create_app(create_config(config), clients)
```

Comment:
weird to pass `config` to `create_config`?

Reply:
Yeah, `create_app_config(config_file)`?